eva-server/src/main/java/uk/ac/ebi/eva/server/Utils.java

```diff
@@ -18,13 +18,29 @@
  */
 package uk.ac.ebi.eva.server;
 
+import org.apache.commons.lang3.StringUtils;
 import org.opencb.datastore.core.QueryOptions;
 import org.springframework.data.domain.PageRequest;
 
 import uk.ac.ebi.eva.lib.repository.VariantEntityRepository;
 
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class Utils {
 
+    private static Map<String, String> apiToMongoDocNameMap;
+
+    static {
+        initApiToMongoDocNameMap();
```

Comment:
The intention could be absolutely clarified by making the method return a value and assign it to the static variable:

```java
static {
    apiToMongoDocNameMap = initApiToMongoDocNameMap();
}
```